### ISSUE TITLE
Fails to run in minetest nightly build.

### ISSUE THREAD

**Issue by KarlHegbloom** [minetest-technic/technic_game#15]
```
...etest/games/technic_game/mods/plantlife/trunks/nodes.lua:330: attempt to index a nil value
stack traceback:
  ...etest/games/technic_game/mods/plantlife/trunks/nodes.lua:330: in main chunk
  [C]: in function 'dofile'
  ...netest/games/technic_game/mods/plantlife/trunks/init.lua:12: in main chunk
```

**Comment by ShadowNinja** [comment]
@VanessaE: Plantlife bug.

**Comment by VanessaE** [comment]
It appears to be a node being called for by trunks mod that is undefined in KarlHegbloom's setup.  As a rough guess, probably the pine tree node from minetest_game.  Is that package up-to-date?

KarlHegbloom, can you please insert `print("tried to make trunk root for "..dump(MoD)..", "..dump(TRuNK))` at plantlife/trunks/nodes.lua line 329, reproduce the crash, and tell me the result?

(copy and paste the above, as the variable names are case-sensitive)

**Comment by KarlHegbloom** [comment]
No you misunderstand. It's obviously happening when the init.lua file is
loaded, and it attempts to override symbols that do not exist because
they've been changed or something. I'm busy today but I will pin this
email, and try and get back to it maybe this evening, when I will follow
your suggestion and send you the result.

Does anyone else have the technic_game working in daily-build minetest?

**Comment by KarlHegbloom** [comment]
Hmmm. The above comment refers to a different bug probably related to the one I pasted that error message to, and that error message might be the result of me having deleted something to try and get it running. I'll begin with a fresh check out and go from there so it'll be a little more helpful. If I recall correctly, it had to do with trying to override little brown 3d mushrooms or something. From flowers.

**Comment by VanessaE** [comment]
Yeah, I didn't notice this was for technic_game (I read it as just "technic", i.e. the mod). My upstream mods are not written for this game, per se, rather for minetest_game.

**Comment by KarlHegbloom** [comment]
I actually added a test for a nil result of the registered_nodes lookup, and it now loads... printing this:

No such node: default:pinetree
No such node: trees:tree_conifer
No such node: trees:tree_mangrove
No such node: trees:tree_palm

... and so now it bombs with:

2015-11-29 19:19:06: ERROR[Main]: ModError: Failed to load and run script from /usr/share/minetest/games/technic_game/mods/mesecons/mesecons_torch/init.lua:
2015-11-29 19:19:06: ERROR[Main]: ...games/technic_game/mods/mesecons/mesecons_torch/init.lua:79: attempt to perform arithmetic on field 'LIGHT_MAX' (a nil value)
2015-11-29 19:19:06: ERROR[Main]: stack traceback:
2015-11-29 19:19:06: ERROR[Main]:   ...games/technic_game/mods/mesecons/mesecons_torch/init.lua:79: in main chunk

It makes me wonder... is this a feature or a bug? Since this is the technic_game, perhaps I'm expected to learn lua and minetest mod coding? 8-)

I wish I had more time for it, honestly. Right now, I've got a very important deadline to meet, so my tiem with Minetest must be restricted to the time I also spend with my son, and we both play.

He turned our cliffside mansion into a waterfall! Fun.
